### DISCUSSION TITLE
Do we have any architectural discussion on Annotator and Reviewer?

### DISCUSSION THREAD

**Discussion by hiq-ayeddula** [Project-MONAI/MONAILabel#1645] [General]
I went through your videos on Youtube 
https://www.youtube.com/watch?v=-HAryYAO5J4
Do we have any architectural discussion on Annotator and Reviewer kind of things?

**Comment by diazandr3s** [comment]
Hi @hiq-ayeddula,

Thanks for asking this.

Here is more information about the MONAI Label reviewer: 

**Video presentation:** https://youtu.be/oudWSBerPuY?si=0_6CGqbviozV2EUn&t=8233
**First PR showing schema and description:** https://github.com/Project-MONAI/MONAILabel/issues/722
**Add MONAILabelReviewer Slicer Extension:** https://github.com/Project-MONAI/MONAILabel/pull/821 and source code: https://github.com/Project-MONAI/MONAILabel/tree/main/plugins/slicer/MONAILabelReviewer

Can I ask in which modality (i.e. CT, MRI) and regions/organs are you trying to use this Reviewer?

I hope this helps.

Let us know,

**Reply by hiq-ayeddula** [reply]
Currently, we are testing out the compatibility of MONAI for one of our custom use cases related to Brain CT.
